### PR TITLE
Add a static landing page for a marketing campaign 

### PR DESCRIPTION
**Pull request by edash22**
Just testing the process of a static html file. Took the home page and cleaned it up a little. Want to be sure that I can view it.

### CODE REVIEW

File: static/chicago-immigration-law.html [hunk 1]
```diff
@@ -0,0 +1,912 @@
...
+        <div class="col-md-8">
+          <form name="contact" method="POST" data-netlify="true">
+            <label for="fname">Name *</label>
+            <input type="text" id="fname" name="name" required="" >
+            <label for="femail">Email *</label>
+            <input type="email" id="femail" name="email" required="" >
+            <label for="fphone">Phone</label>
+            <input type="text" id="fphone" name="phone" >
+            <label for="fmsg">Message *</label>
+            <textarea id="fmsg" name="message"></textarea>
```
Comment (jonkeane) [inline static/chicago-immigration-law.html:881]:
```suggestion
            <textarea id="fmsg" name="message" required=""></textarea>
```

This one is required as well? Or should we take off the `*` from the label?

**Comment by jonkeane** [comment]
Thanks for this @edash22 Do we need both the chicago-immigration-law.html page and landingTemp.html page when we go live? Or just the first one? It also looks like the image [439616480.jpg](https://github.com/SouthamLaw/southamlaw-site/pull/53#diff-1f1884d364ce6896f89f98f65aef6eebfed4bc3201e63ebf814bf58311c4bb2f) isn't used in the real site as well, yeah?

We don't need to do this now (and I can do this on my own later), but it would be good to have the CSS in a separate CSS file, yeah?

**Comment by jonkeane** [comment]
Ok, I've confirmed with Keith this looks good. I'll merge it now (and it will then be live). We can of course make changes if we need to with the same PR process here.

**Comment by edash22** [comment]
Thank you Jon,

 

I’ve been traveling and out of touch! Thanks for taking care of everything and fixing the phone number.

 

I’m on vacation through next Wednesday, but checking emails in the evening/early morning.

I don’t imagine anything will come up that you can’t handle though.

 

I appreciate your help and support!

 

Kind regards,

Ellen

**Pull request merged**